app/templates/events/showUpcomingEvents.html

```diff
@@ -0,0 +1,23 @@
+<table class="table">
+
+  {% if upcomingEvents|length > 0 %}
+    <thead class="tableHeader">
+      <th scope="col"><h4><strong>Program</strong></h4>
+      <th scope="col"><h4><strong>Event Name</strong></h4>
+      <th scope="col"><h4><strong>Start Date</strong></h4>
+      <th scope="col"><h4><strong>End Date</strong></h4>
+        </thead>

```

Comment:
indentation of the closing tag should match the opening one

Reply:
fixed